server/engine/src/main/antlr4/org/eclipse/lsp/cobol/core/parser/Db2SqlParser.g4

```diff
@@ -1699,8 +1723,8 @@ dbs_xml_table_ordinality_column_defn: dbs_column_name FOR ORDINALITY;
 dbs_collection_derived_table :  UNNEST LPARENCHAR (dbs_ordinary_array_expression (dbs_comma_separator dbs_ordinary_array_expression)* | dbs_assosiative_array_expression) RPARENCHAR (WITH ORDINALITY)? dbs_correlation_clause?;
 dbs_ordinary_array_expression : IDENTIFIER;
 dbs_assosiative_array_expression : NONNUMERICLITERAL;
-dbs_joined_table : (dbs_table_reference (INNER | (LEFT | RIGHT | FULL) OUTER?)? JOIN dbs_table_reference ON dbs_join_condition | dbs_table_reference CROSS JOIN dbs_table_reference | LPARENCHAR dbs_joined_table RPARENCHAR);
 dbs_join_condition: dbs_inner_left_outer_join | dbs_full_join_expression;
+dbs_join_type: (INNER | (LEFT | RIGHT | FULL) OUTER?);
```

Comment:
`dbs_join_type` only used in one place, is there a reason not to put it right there?

Reply:
Oh, I see. It is because of the comment. Maybe then at least move it where it is used to be next to it?

Reply:
No specific reason. I will remove this rule

Reply:
I removed it. Comment is just for help